lib/cib/cib_attrs.c

```diff
@@ -86,10 +93,12 @@ find_nvpair_attr_delegate(cib_t * the_cib, const char *attr, const char *section
     }
 
     xpath_string = calloc(1, xpath_max);
-    offset += snprintf(xpath_string + offset, xpath_max - offset, "%.128s", get_object_path(section));
```

Comment:
Btw. `path_max - offset < 0` will never hold true when cast to unsigned type (`size_t`).